doc/users_guide.rst

```diff
@@ -84,6 +84,27 @@ access to many LISA related functions (done automatically by Vagrant)::
 
 .. tip:: Run ``lisa-help`` to see an overview of the provided LISA commands.
 
+Running tests
```

Comment:
So I'm torn between having this here or in the kernel testing section.

Maybe we could have a "LISA use cases" section in the users guide that briefly points to kernel tests / trace analysis?

Reply:
Initially, I had that paragraph in the workflows.rst introduction. After that, I realized that the user guide provided no clue about running the tests. So the reasoning was: if you just look at the user guide, you get the quickest path to running tests in the simplest way possible. This way the user-guide is self-contained, especially for somebody coming to LISA the first time and who just wants to get a feeling of what it can do.

If you plan on using LISA a bit more, then you are probably ready to read a bit more to see what kind of workflow suits your need. Maybe it would make sense to have a toplevel "LISA use cases" page, with a subpage for exekall/bisector and another one for notebooks.